### ISSUE TITLE
Find someone to take over QST

### ISSUE THREAD

**Issue by sieren** [sieren/QSyncthingTray#247]
Since I have limited time to take care of QST, I'd like for someone to step in and take over leading this project.

If anyone is interested, please drop me a line here or send an email to matthias (at) s-r-n.de

**Comment by es20490446e** [comment]
Are you using the software yourself?

**Comment by sieren** [comment]
Not anymore, I'm using the official tray icon that comes with the macOS bundle.

**Comment by es20490446e** [comment]
Opened a [topic on the syncthing forum](https://forum.syncthing.net/t/adopting-qsyncthingtray/13844).

**Comment by sieren** [comment]
Awesome, thanks for taking the initiative! 
I wish I could work more on this, but I simple dont have the time and I figured https://github.com/Martchus/syncthingtray is by far the superior and better cross-platform tool by now.

**Comment by es20490446e** [comment]
Sorry, but I like your software more 😛

That one is over-featured.

**Comment by Martchus** [comment]
In theory it would be possible to add a simplified GUI to my Syncthing Tray. But I've created it mainly because QSyncthingTray lacked a lot of features I like. Still, I'm wondering what's actually bothering you. If it is the dependency for Qt WebEngine/WebKit: You can disable it at build time.

**Comment by es20490446e** [comment]
@Martchus When I judge the usefulness of a software I do through the eyes of someone not very tech savvy. On one hand because I want it to be usable to as much people as possible, on the other because the amount of options a software has tells something about how much work there has been in simplifying its internals.

The more streamlined a software is the less technical options it has, cause the software is able to make those decisions automatically and because it is selective in what to include and exclude. The software only provides options related to the user work itself, but not about the technical aspects of it. The simpler the API and interface is the more advance the internals usually are.

The oposite is wanting the software to do everything and cover any possible situation, by making the user decide about corner cases that will rarely happen. Aka [90% of the time it works every time](https://youtu.be/pjvQFtlNQ-M).

When I develop software it usually has very few features, very little code, and very few tools. But what people don't see is how much improvement took me to reach that point, that so little does the work perfectly.

Is not what I have left in, is what I have been able to left out without that having consequences.

**Comment by es20490446e** [comment]
@sieren I could maintain qsyncthingtray if it's only on Linux. I would delay a little bit in the beginning, buy once I finish what I'm working on I would be able to properly maintain it.

**Comment by Martchus** [comment]
Ok, so you don't like the "advanced" configuration options. These are definetly something I'm not going to simplify so I guess Syncthing Tray will never be an alternative for you then. Considering these different preferences it is actually good that I decided against contributing to QSyncthingTray and rather started my own project.

I don't quite get why you think Syncthing Tray would let the user decide about corner cases, though. The options of Syncthing Tray are mostly about enabling/disabling features you want or don't want to use and about tweaking the appearance to your taste. The configurable intervals might be a bit odd, indeed. However, QSyncthingTray has this kind of configuration, too.

I also disagree on the "few feature" philosophy in general. You will for sure end up with something that works perfectly (or at least quite well). However, the scope of that program will be rather limited, too. For instance QSyncthingTray does not help me to quickly pause/unpause a device or directory simply because that feature is missing.

But it is still nice that you might maintain QSyncthingTray. Syncthing's API and Qt's API are very stable in my experience so that is actually not going to be that much work - especially since QSyncthingTray only uses a small fraction of these APIs.

**Comment by es20490446e** [comment]
@Martchus 
> QSyncthingTray does not help me to quickly pause/unpause a device or directory

What's the purpose of pausing?

**Comment by Martchus** [comment]
I can tell you my use cases: I usually use it if I have only a limited internet connection and therefore want temporarily exclude a (large) directory completely. I occasionally use it to prevent rescans of large but rarely changing directories completely on slow devices (setting the rescan interval to zero does not prevent the initial rescan on startup).

**Comment by es20490446e** [comment]
@sieren Could you at least for the time being make a release of the current code, so downstream packages can update to it?

**Comment by jcrben** [comment]
@sieren seems like the latest version of syncthing handles mac pretty well natively, should this project be archived? or at least point out that people can just install syncthing and run it?

**Comment by sieren** [comment]
@jcrben yeah, I've actually moved on to the mac version myself. We have a few Linux users in the community though, but there are other projects worth using in the meantime (@Martchus 's https://github.com/Martchus/syncthingtray for example) that are much more extensive and better written (this was my first Qt project ever).

I'll consider sunsetting this for good or at least updating the README and pointing people to better alternatives.

**Comment by es20490446e** [comment]
I think those alternatives suck.

I know how this ends: me eventually coding the alternative.
